packages/runtime/src/components/Diagnostic.svelte

```diff
@@ -68,7 +68,7 @@
   }
 
   .message-item {
-    border-bottom: 1px dotted #666;
+    border-bottom: 2px dashed #838383;
```

Comment:
I think we can keep the style as before as it might looks too attractive.

Reply:
Done